### PR TITLE
Add custom environment variable passthrough

### PR DESCRIPTION
**Pull request by zazab**
Can be used for passing pkgrel in PKGBUILD:

```
...
pkgrel=${REL-1}
...
```

`curl 'http://saturated/v1/build/http://repo.path?environ=REL=2'`

### CODE REVIEW

**Comment by seletskiy** [comment]
Can you make possible to pass several environment variables, not just one?

**Comment by zazab** [comment]
add multiple evironment variables support

File: main.go [hunk 1]
```diff
@@ -142,13 +142,24 @@ func (handler *BuildHandler) ServeHTTP(
 
 	fmt.Fprintf(topLevelLogger, "running build task for '%s'", repoUrl)
 
-	err := runBuild(
+	err := request.ParseForm()
+	if err != nil {
+		errorMessage := fmt.Sprintf("error parsing request: %s", err)
+		fmt.Fprintf(topLevelLogger, errorMessage)
+		response.WriteHeader(http.StatusBadRequest)
+		response.Write([]byte(errorMessage))
```
Comment (seletskiy) [inline main.go]: `topLevelLogger` will already replicate all messages to the `response` (see declaration above). So, `WriteHeader` should be called sooner and `reponse.Write` should be removed.

**Comment by seletskiy** [comment]
Thanks for the contribution!

**Pull request merged**